app/routers/getjoke.py

```diff
@@ -0,0 +1,10 @@
+import requests
+from fastapi import APIRouter
+
+
+router = APIRouter()
+
+
+@router.get("/getjoke")
+def getjoke():
+    return requests.get('http://api.icndb.com/jokes/random').json()
```

Comment:
Can you rewrite it to be on the frontend side?
Every HTTP call from the server side might majorly degrade our performance.

Reply:
yes.

Reply:
I did